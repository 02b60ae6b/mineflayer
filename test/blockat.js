mf.setTimeout(function() {
    mf.print("Test timed out.\n");
    mf.exit();
}, 5000);

mf.onChunkUpdated(function(start, size) {
    if (start.x === 0 && start.y === 0 && start.z === -80) {
        var pt = new mf.Point(0,0,0);
        for (pt.y = 58; pt.y <= 62; pt.y++) {
            for (pt.z = start.z; pt.z < start.z+5; pt.z++) {
                for (pt.x = start.x; pt.x < start.x+5; pt.x++) {
                    mf.print(mf.blockAt(pt).type + ",");
                }
                mf.print("\n");
            }
            mf.print("\n");
        }
        mf.exit();
    }
});
