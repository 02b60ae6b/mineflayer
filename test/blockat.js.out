16,24,1,24,1,
16,24,1,24,1,
1,1,1,3,3,
3,3,24,3,3,
3,3,3,3,3,

16,24,24,12,24,
1,24,1,24,1,
1,1,1,24,1,
3,3,24,3,3,
3,3,3,3,24,

1,12,12,12,12,
1,12,1,24,12,
1,24,24,24,1,
1,1,24,24,1,
3,3,1,1,24,

12,12,12,12,12,
12,12,12,12,12,
1,12,24,24,12,
1,24,12,24,12,
1,1,1,1,24,

12,12,12,12,12,
12,12,12,12,12,
12,12,12,12,12,
3,12,12,12,12,
3,3,1,1,12,

